journalbeat/Dockerfile

```diff
@@ -4,9 +4,11 @@ MAINTAINER Noémi Ványi <noemi.vanyi@elastic.co>
 RUN set -x && \
     apt-get update && \
     apt-get install -y --no-install-recommends \
-      python-pip virtualenv libsystemd-dev libc6-dev-i386 gcc-arm-linux-gnueabi && \
+      python-pip virtualenv python3 libsystemd-dev libc6-dev-i386 gcc-arm-linux-gnueabi && \
```

Comment:
Is `python3-pip` not needed to install Python 3 packages?

Reply:
It seems that virtualenv is smart enough to install the proper version of pip in the virtual environments.